src/runner.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, configs, args):
 
     def _create_node(self, tarball_node, plan_config):
         node = {
-            'parent': tarball_node['parent'],
+            'parent': tarball_node['_id'],
```

Comment:
So the tarball will be the parent of test nodes. I don't understand it thoroughly. This will be required when we have multiple tarball nodes of the same checkout node. That we do not have atm. What are the other possible child nodes of checkout?

Reply:
I don't know, there's just one checkout and one tarball node at the moment for each kernel revision.  We'll update things if needed as the pipeline evolves.